source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch} from '../github-helpers';
+
+function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
+	dropdown.classList.add('rgh-git-checkout'); // Mark this as processed
+	const [, user, repository] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
+
+	const tabContainer = select('tab-container', dropdown)!;
+	tabContainer.style.width = '370px';
```

Comment:
This causes a scrollbar that hides it if CodeSpaces is active

<img width="399" alt="Screen Shot 2020-09-26 at 20 09 42" src="https://user-images.githubusercontent.com/1402241/94353240-4cb28780-0034-11eb-99ac-7c123d40d024.png">

Limit our `pre` instead, maybe with `fit-content` or whatever will fill the width of the popup without increasing it.